examples/mass_spring_3d.py

```diff
@@ -1,18 +1,18 @@
 import taichi as ti
+import numpy as np
 try:
     import taichi_glsl as tl
     import taichi_three as t3
 except ImportError:
     print('This example needs the extension library Taichi THREE to work.'
           'Please run `pip install --user taichi_three` to install it.')
+assert t3.__version__ == (0, 0, 3)
```

Comment:
Why do we need to assert the version of `taichi_three` here?

If the `taichi_three` updated this example maybe will be inavaliable ;)

Reply:
I guess Taichi THREE v0.0.4 will break the API again, if not, we're OK to remove this before v0.0.4 release.